README.md

```diff
@@ -0,0 +1 @@
+Hello! I am MukhammadSultan and I don't know what to do!
```

Comment:
Why do you use print inside this function! Please remove unneccessary lines of code

Reply:
I will remove it rn